py-polars/polars/datatypes/classes.py

```diff
@@ -342,6 +378,8 @@ class Decimal(NumericType):
         It may be changed at any point without it being considered a breaking change.
     """
 
+    __slots__ = ("precision", "scale")
+
```

Comment:
Nit: remove the blank line here to be consistent with other slots implementations:
```suggestion
    __slots__ = ("precision", "scale")
```